code/game/objects/structures/crates_lockers/closets/secure/security.dm

```diff
@@ -346,6 +346,9 @@
 		new /obj/item/weapon/gun/projectile/revolver/detective(src)
 		new /obj/item/taperoll/police(src)
 		new /obj/item/clothing/accessory/holster/armpit(src)
+		new /obj/item/clothing/glasses/sunglasses/noir(src)
+		new /obj/item/clothing/glasses/sunglasses/noir(src)
+		new /obj/item/device/flashlight/seclite(src)

```

Comment:
Probably better to just spawn a single pair on the detective, and have none in the locker to make it a bit more unique.